components/python/mergedeep/test/results-all.master

```diff
@@ -0,0 +1,18 @@
+Warning: 'classifiers' should be a list, got type 'tuple'
+running test
+WARNING: Testing via this command is deprecated and will be removed in a future version. Users looking for a generic test entry point independent of test runner are encouraged to use tox.
+running egg_info
+writing mergedeep.egg-info/PKG-INFO
+writing dependency_links to mergedeep.egg-info/dependency_links.txt
+writing top-level names to mergedeep.egg-info/top_level.txt
+reading manifest file 'mergedeep.egg-info/SOURCES.txt'
+reading manifest template 'MANIFEST.in'
+warning: no previously-included files matching '*test_*.py' found anywhere in distribution
+adding license file 'LICENSE'
+writing manifest file 'mergedeep.egg-info/SOURCES.txt'
+running build_ext
+
+----------------------------------------------------------------------
+Ran 0 tests in 0.000s
+
```

Comment:
0 tests have been run. Using tox is recommended.

Reply:
Yes.  The tox recommendation is generic message displayed always by `setup.py` because testing via `setup.py` is considered deprecated.  But, the testing using another tools (like `tox`) must be actively supported by the project (`setup.py test` is supported always, by default), which is not the case here with `mergedeep`, because `mergedeep` does not support any testing AFAIK (which is pretty common for python projects as I learned recently), so `setup.py test` just does the best it can, which is to report `Ran 0 tests`.

Anyway, I just noticed there is timing info at that line so I'll remove it to have test results normalized (via a separate PR).

Once I encounter first project that supports tox testing, I'll implement support for tox into our tools/makefiles.